src/fileutil.c

```diff
@@ -102,22 +102,27 @@ char* find_file(const char* filename)
 {
     char tmppath[MAX_PATH];
     tmppath[0] = '\0';
+    const char *filepathptr = tmppath;
     expand_user_path(filename, tmppath, sizeof(tmppath));
+    if(tmppath[0] == '\0')
+    {
+        filepathptr = filename;
+    }
```

Comment:
Please don't start doing enters before the opening curly:

https://github.com/SIPp/sipp?tab=readme-ov-file#contributing

And a space between `if (`.

And don't do:
```
{ // comment here
```
Please look at the surrounding code and format the style in a similar fashion.

Reply:
In the entire PR please :)

Reply:
Thanks for reviewing. I will test all scenarios again (as I have made some fixes based on your previous comments that I didn't understand then) and remind you.

Reply:
Hi @wdoekes I have finished testing. Please let me know if any more issues or room for improvement or merge it as you seem fit. Thanks!